pages/querying/functions.mdx

```diff
@@ -157,12 +157,70 @@ This section contains the list of built-in query modules and their procedures.
 
 ### Schema
 
-The `schema` procedures return information about the database schema.
+The `schema` procedures return information about the database schema and can be used to assert constraints and indices exist in the database.
 
 <Callout type="info">
 You can execute these procedures on [graph projections, subgraphs or portions of the graph using the `project` function](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph).
 </Callout>
 
+{<h4> <code> schema.assert() </code> </h4>}
+
+The `schema.assert()` is used to assert indices and constraints exist in Memgraph.
```

Comment:
Something is wrong with this sentence

Reply:
assert existence of indices and constraints.